app/javascript/components/App.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import Greeting from './Greeting';
+
+const App = () => {
+  return (
+      <Greeting />
+  );
+};
```

Comment:
- [ ] Great Job working on displaying the messages but I think it would be great if you add a reducer and store and to use redux to link your API with your frontend component since it is mentioned in the requirements to add them so kindly add a reducer and store and wrap your app with the store to make it works as expected 🔥🙏🏻💪🏻✨

![image](https://github.com/demesameneshoa/hello-rails-react/assets/58553711/02367e41-878f-49f5-9ad4-b0f2261ef6e5)